omod/src/main/java/org/openmrs/module/locationbasedaccess/web/controller/ModuleDependencyController.java

```diff
@@ -0,0 +1,51 @@
+package org.openmrs.module.locationbasedaccess.web.controller;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.openmrs.module.Module;
+import org.openmrs.module.ModuleFactory;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.BaseRestController;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/lbac/module-dependency")
+public class ModuleDependencyController extends BaseRestController {
+
+	/**
+	 * @return modules required for LBAC module installation
+	 */
+	@RequestMapping(method = RequestMethod.GET)
```

Comment:
Can't we use getMapping?
https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/bind/annotation/GetMapping.html

Reply:
@suthagar23  we can use if we update the spring version to 4.3